calculator.py

```
@@ -1,15 +1,31 @@
 import math
+import random
```

Ruff

`random` imported but unused

Remove unused import: `random`

(F401)